typings/nuxt__types/index.d.ts

```diff
@@ -1,8 +1,10 @@
 import '@nuxt/types'
 import type { Details as UADetails } from 'express-useragent'
+import type { NuxtI18nApi } from '@nuxtjs/i18n/types/vue'
 
 declare module '@nuxt/types' {
   export interface NuxtAppOptions {
     $ua: UADetails | null
+    localePath: NuxtI18nApi.localePath
```

Comment:
Huh, this should already be on this interface due to the `@nuxtjs/i18n` being included in the `tsconfig.json` `types` option :thinking:

Reply:
Ah, I probably added it before adding `@nuxtjs/i18n`, OR simply because the issue said to add it 😁